File: src/js/fizzbuzz.js
for (var i = 1; i <= 100; i++) {
    if (i % 3 == 0 && i % 5 == 0)
        process.stdout.write("FizzBuzz");
    else if (i % 3 == 0)
        process.stdout.write("Fizz");
    else if (i % 5 == 0)
        process.stdout.write("Buzz");
    else
        process.stdout.write(i.toString());


    if (i % 10 == 0)
        process.stdout.write("\n");
    else
        process.stdout.write(",");
}

File: src/scala/fizzbuzz.scala
object FizzBuzz extends App {
    (1 until 100).map(i => (i % 3, i % 5) match {
        case (0, 0) => "FizzBuzz"
        case (0, _) => "Fizz"
        case (_, 0) => "Buzz"
        case _ => i}).zipWithIndex.foreach {
            case (i, index) => {
                print(s"$i,")
                if ((index + 1) % 10 == 0)
                    println
            }
        }
}